student-guide/_bookdown.yml

```diff
@@ -1,11 +1,12 @@
-book_filename: "Student_Guide" # Do not change
```

Comment:
@KatherineCox Ok if we add the # Do not change note to AnVIL_Template? If we change the book name, the docx download link (in the pull request action comment) breaks.

Reply:
Sounds reasonable to me 👍